docs/requirements.txt

```diff
@@ -3,3 +3,5 @@
 # SPDX-License-Identifier: Unlicense
 
 sphinx>=4.0.0
+adafruit_circuitpython_typing>=1.8.2
```

Comment:
This should be pinned as `~=1.5` since that's when `pwmio` was added to it.  This also prevents it from being over restrictive.

Reply:
Thx, altered it.